pkgs/by-name/sl/slicer3d/package.nix

```diff
@@ -0,0 +1,133 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+  hwloc,
+  fetchurl,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
```

Comment:
The latest stable release is now `5.8.1`. I also tried a recent pre-release build to check if the need for `hwloc_old` override subsided, but still no as of writing.

```nix
  version = "5.9.0";
...
  src = fetchzip {
    name = "Slicer-${version}-linux-amd64.tar.gz";
    url = "https://download.slicer.org/bitstream/68cb96fe903cebf926bad01b";
    hash = "sha256-hNHfSs2v75rofnlyGslHmpJYXMRglQfE7I2y8+mTyfM=";
    extension = "tar.gz";
  };
```